packages/backend/src/database/migrations/20220516074540_add_user_to_jobs.ts

```diff
@@ -0,0 +1,22 @@
+import { Knex } from 'knex';
+
+const COMPILE_JOB_TABLE_NAME = 'jobs';
+
+export async function up(knex: Knex): Promise<void> {
+    const [existingUserUuid] = await knex
+        .table('users')
+        .select('user_uuid')
+        .limit(1);
+    await knex.schema.alterTable(COMPILE_JOB_TABLE_NAME, (tableBuilder) => {
+        tableBuilder
+            .uuid('user_uuid')
+            .notNullable()
+            .defaultTo(existingUserUuid ? existingUserUuid.user_uuid : null);
```

Comment:
This will make all future jobs belong to the first user unless another user is specified, is that intentional?

If we just made this null would it be a problem?

Reply:
Yes, that was discussed here https://github.com/lightdash/lightdash/issues/2107#issuecomment-1127346303 let me know what you think

Reply:
It's ok to backfill the values with a user id but this will also make future inserts default to the existing user id. It's baked into the table definition.

I think it's fine as nullable, but if you want it notNullable - then you need to manually insert values rather than using the defaultTo